tools/rosbag/src/player.cpp

```diff
@@ -448,10 +445,10 @@ void Player::waitForSubscribers() const
     bool all_topics_subscribed = false;
     std::cout << "Waiting for subscribers." << std::endl;
     while (!all_topics_subscribed) {
-        all_topics_subscribed = true;
-        foreach(const PublisherMap::value_type& pub, publishers_) {
-            all_topics_subscribed &= pub.second.getNumSubscribers() > 0;
-        }
+        all_topics_subscribed = std::all_of(std::begin(publishers_), std::end(publishers_),
```

Comment:
Please wrap right after the first open parenthesis (`std::all_of(\n`) to avoid the aligned indendation of the following lines. All arguments can then be one level indented.